README.md

```diff
@@ -51,7 +51,22 @@ By default, the Datadog trace id gets automatically injected into the logs for c
 
 Set the environment variable `DD_LOGS_INJECTION` to `false` to disable this feature.
 
-## Custom logger
+## Handler Wrapper
+
+In order to instrument individual invocations, the Datadog Lambda library needs to wrap around your Lambda handler function. This is normally achieved by pointing your function's handler setting to the provided Datadog handler function and passing the original handler function through an environment variable to be called by the Datadog handler.
+
+If the method described above doesn't work for you, instead of overriding the handler and setting the `DD_LAMBDA_HANDLER` environment variable, alternatively you can apply the Datadog Lambda library wrapper in your function code like below:
```

Comment:
```suggestion
If this method doesn't work for you, instead of overriding the handler and setting the `DD_LAMBDA_HANDLER` environment variable, you can apply the Datadog Lambda library wrapper in your function code like below:
```